core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchWriter.java

```diff
@@ -219,7 +223,7 @@ public void run() {
             updateUnknownErrors("Max latency task failed " + t.getMessage(), t);
           }
         }
-      }, 0, this.maxLatency / 4);
+      }, 0, this.maxLatency / MAX_LATENCY_DIVISOR);
```

Comment:
Why is checking for exceeding maxLatency better to do 16 times per interval instead of 4? This seems like it could be bad, because it synchronizes the batch writer more frequently, and it's not clear what is gained in return.

Reply:
I am just thought that speeding up the polling would also increase throughput.  That was the intent.

Reply:
It's an interesting idea, but I'm not sure what impact the object locking will have on the outcome. Have you run any controlled tests with various maxLatency settings before and after this change?